### ISSUE TITLE
Inaccessible Fancy Field

### ISSUE THREAD

**Issue by GeorgeTaveras1231** [xogroup/union#307]
The `Field` and `Dropdown` components in `@xo-union/tk-component-fancy-field` are inaccessible to screen-readers because they do not have a `label` associated to them.

Solution
===
- Assign an id to the `input` element ( Use [`#fieldId`](https://github.com/xogroup/union/blob/master/packages/%40xo-union/tk-component-fields/src/utilities/index.js#L9-L15) utility from `@xo-union/tk-component-fields` )
> NOTE: the fieldId function may need to be duplicated if it continues to use the `__webpack_hash__` helper, because this `hash` should only change when the given package changes.
> Additionally, consider changing the uniqueness strategy to use a global counter: something like `(window || global).__UNION_GLOBAL_COUNTER__` and remove the usage of `__webpack_hash__`
> Another option would be to bring back `shortid` (But I'd rather not include that dependency 😄 )

- Create a `label` with, passing the generated id as the `htmlFor` prop.
- Make bootstrap's `.sr-only` class available in the Union ecosystem
> This should be a file added to `tk-css-utilities/src/screenreaders.scss` which should load `bootstrap/scss/utilities/screenreaders`
- Assign the `sr-only` class to the label
